app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TweetsController < ApplicationController
+
+  get "/tweets" do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :"/tweets/tweets"
+    else
+      redirect "/login"
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :"/tweets/create_tweet"
+    else
+      redirect "/login"
+    end
+  end
+
+  post '/tweets' do
+    if !params[:content].empty?
+      @tweet = Tweet.create(params)
+      @tweet.user = current_user
+      @tweet.user = User.find_by_id(session[:user_id])
+      @tweet.save
+      redirect "/tweets/#{@tweet.id}"
+    else
+      redirect '/tweets/new'
+    end
+  end
+
+
+  get "/tweets/:id" do
+    if logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
+      erb :"/tweets/show_tweet"
+    else
+      redirect "/login"
+    end
+  end
+
+
+  get "/tweets/:id/edit" do
+    if logged_in?
+      @tweet = Tweet.find_by(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?